tests/macsec/test_interop_protocol.py

```diff
@@ -99,8 +135,16 @@ def check_bgp_established(ctrl_port, up_link):
 
         # Ensure the BGP sessions have been established
         for ctrl_port in list(ctrl_links.keys()):
-            assert wait_until(BGP_TIMEOUT, 5, 0,
-                              check_bgp_established, ctrl_port, upstream_links[ctrl_port])
+            assert wait_until(
+                BGP_TIMEOUT, 5, 0,
+                check_bgp_established, ctrl_port, upstream_links[ctrl_port]
+            ), (
+                "BGP session did not establish within the specified timeout for port '{}'. "
+                "Current status: '{}'.".format(
+                    ctrl_port,
+                    check_bgp_established(ctrl_port, upstream_links[ctrl_port])["status"]
```

Comment:
check_bgp_established just return True or False depends bgp state is establised or not. It's not a status string. Suggest that it's enough to print out upstream_links[ctrl_port]